ops/ccs-ops-misc/load_test/runtests.py

```diff
@@ -91,48 +99,60 @@ class configData: pass
             print(helpString)
             sys.exit()
         elif opt == "--homePath":
-            configData.homePath = arg
+            configData["homePath"] = arg
         elif opt == "--clientCertPath":
-            configData.clientCertPath = arg
+            configData["clientCertPath"] = arg
         elif opt == "--databaseUri":
-            configData.dbUri = arg
+            configData["dbUri"] = arg
         elif opt == "--testHost":
-            configData.testHost = arg
+            configData["testHost"] = arg
+        elif opt == "--configPath":
+            configData["configPath"] = arg
         elif opt == "--serverPublicKey":
-            configData.serverPublicKey = arg
+            configData["serverPublicKey"] = arg
         elif opt == "--testRunTime":
-            configData.testRunTime = arg
+            configData["testRunTime"] = arg
         elif opt == "--maxClients":
-            configData.testNumTotalClients = arg
+            configData["testNumTotalClients"] = arg
         elif opt == "--clientsPerSecond":
-            configData.testCreatedClientsPerSecond = arg
+            configData["testCreatedClientsPerSecond"] = arg
         elif opt == "--testFile":
             testFile = arg
         elif opt == "--workerThreads":
             workerThreads = arg
         elif opt == "--resetStats":
-            configData.resetStatsAfterClientSpawn = True
+            configData["resetStatsAfterClientSpawn"] = True
         else:
             print(helpString)
             sys.exit()
 
+    ## Read the specified configuration file and discard any falsey key
+    storedConfigData = {k: v for k, v in config.load_from_path(configData["configPath"]).items() if v}
```

Comment:
I get a KeyError here if I run without a configPath set, but configPath isn't listed as a required field. We should do one or the other -- my opinion is, it should probably not be required.

Reply:
Good catch! That's definitely a mistake on my end. I intended it to be optional, I just didn't catch that I was reading from `configData` prior to `configData["configPath"]` being set by merging with `defaultConfigData`. 

I've fixed this so that we use `Dict.get(...)` to fallback to `defaultConfigData["configPath"]` if `--configPath` was not passed-in. I've tested this locally and `--configPath` is now optional as expected.